src/grisp_connect.app.src

```diff
@@ -21,6 +21,7 @@
         {port, 443},
         {connect, true}, % keeps a constant connection with grisp.io
         {ntp, false}, % if set to true, starts the NTP client
+        {ntp_servers, ["0.europe.pool.ntp.org"]},
```

Comment:
We document all `grisp_connect` options in the Readme file, make sure this and others are up to date.